tests/copyright/policy

```diff
@@ -22,6 +22,6 @@ modules/native
  GPLv2+_SSL
 tests
 modules/java/(tools|[^/]*$)
-modules/java-modules/(dummy|elastic|hdfs|http|kafka|[^/]*$)
+modules/java-modules/(dummy|elastic|elastic-v2|hdfs|http|kafka|[^/]*$)
```

Comment:
Yes, we do need to add each one when using the current format. This is because we have both GPL and non-GPL modules in the same folder, and I don't want to add overlapping patterns and row based prioritization, because that can bite us very quickly without ever noticing. As a side note, currently, overlapping declarations produce an error (e.g., the containing folder under GPL and subfolders under LGPL).

After these improvements are merged, this check (and Travis) should always tell you if you forgot to add a module and it should also point out which one. The omission was on my part that I forgot to check `.java` files, so that's why you didn't get a notification about it before.